addons/storyshots/src/angular/renderTree.js

```diff
@@ -1,4 +1,6 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
How are people expected to get `jest-preset-angular` dependency? As contrary to `@angular/core`, it's by no means guaranteed to present in an angular project